### PR TITLE
Express

### LINKED ISSUE #3: Get all
get all items in store

### LINKED ISSUE #4: get specific
get info about specific item

### LINKED ISSUE #5: post item
create new item

### LINKED ISSUE #6: put item
update item

### PR DESCRIPTION
**Pull request by MonserratGenereux**
Closes #3
Closes #4 
Closes #5
Closes #6

### CODE REVIEW

File: app/models/responsable.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ const mongoose = require('mongoose');
 module.exports = mongoose.model("Responsable",{
     name: String,
     age: Number,
-    gender: char,
+    gender: String,
     address: String,
     phone: Number,
-    MemberSince: String
+    MemberSince: String,
```
Comment (dtoledom) [inline app/models/responsable.js]: Change it to `memberSince`

File: app/models/stores.js [hunk 2]
```diff
@@ -4,5 +4,6 @@ module.exports = mongoose.model("Stores",{
     name: String,
     location: String,
     responsable: String,
-    phone: Number
+    phone: Number,
+    responsable: String
```
Comment (dtoledom) [inline app/models/stores.js]: This field is repeated. Erase one.

File: router.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+let router = require('express').Router();
```
Comment (dtoledom) [inline router.js:1]: Use two spaces instead of tab characters in the entire file. You can configure your text editor to do that.

File: router.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  Items.findById(req.params._id, function(err, items){
+    if(!err) res.send(items);
+	else console.log ('ERROR: ' + err);
+	});
+};
+
+//POST
+addItems = function(req,res){
+	console.log('POST');
+	console.log(req.body);
```
Comment (dtoledom) [inline router.js]: Delete this console logs.

File: server.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const api = require('./router');
+
+//Schemas
```
Comment (dtoledom) [inline server.js]: These schemas are not used anywhere. Don't import stuff you are not using.

File: server.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    next();
+});
+
+//REST functions
+app.use('/api', api);
+
+/////////////////////////////////////
+
+//Connection
+mongoose.connect("mongodb://localhost:27017/ProyectoMoviles", function(err,db){
```
Comment (dtoledom) [inline server.js:26]: Add an issue to handle this hardcoded connection url in a better way.

File: server.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    if(!err){
+        console.log("we are connected to mongo");
+    }
+})
+
+app.get('/', function(req, res){
+	res.send('hola mundo');
```
Comment (dtoledom) [inline server.js]: Use another message for the Get request to the root. Somethin like `Moviles API`

File: server.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+});
+const server = app.listen(port, function(){
+    //Response of try to start server
+    console.log("Listening on port", server.address().port )
+})
```
Comment (dtoledom) [inline server.js]: Add new line.

**Review by dtoledom — CHANGES_REQUESTED** [review]

File: server.js (hunk 8 above)
Comment (MonserratGenereux) [reply server.js]: listo 💃

File: router.js (hunk 3 above)
Comment (MonserratGenereux) [reply router.js:1]: Ya 😃

**Review by dtoledom — APPROVED** [review]
LGTM

**Pull request merged**